sdk/core/azure-core-http-httpurlconnection/src/main/java/com/azure/core/http/httpurlconnection/HttpUrlConnectionClient.java

```diff
@@ -0,0 +1,160 @@
+package com.azure.core.http.httpurlconnection;
+
+import com.azure.core.http.*;
+import com.azure.core.util.Context;
+import reactor.core.publisher.Mono;
+
+import java.io.*;
+import java.net.*;
+import java.nio.charset.StandardCharsets;
+
+public class HttpUrlConnectionClient implements HttpClient {
+
+    private HttpRequest request;
+    private HttpURLConnection connection;
+
+    // Default constructor
+    public HttpUrlConnectionClient(){}
```

Comment:
We shouldn't have a public constructor - everything should be instantiated through the builder.